### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,113 +1,108 @@
...
-      fastercsv
       nokogiri
     bagit (0.3.2)
       docopt (~> 0.5.0)
       validatable (~> 1.6)
-    blue-daemons (1.1.11)
-    bluepill (0.1.1)
-      activesupport (>= 3.2, < 5)
-      blue-daemons (~> 1.1)
+    bluepill (0.0.64)
```
Comment (drh-stanford) [inline Gemfile.lock]: for some reason, bluepill got downgraded to 0.0.64. the current version is 0.1.2.

Comment (cbeer) [inline Gemfile.lock]: Related discussion at https://github.com/sul-dlss/item-release/pull/68#issuecomment-255808000

**Comment by peetucket** [comment]
i just tried upgrading to dor-services 5.13 in this branch locally and it updated activefedora to 8.2.1

is that ok?  all tests pass

**Comment by cbeer** [comment]
You'll probably want to merge https://github.com/sul-dlss/common-accessioning/pull/76 first.

**Comment by drh-stanford** [comment]
Added a bluepill pin to `master` for the time being... see https://github.com/sul-dlss/item-release/pull/68#issuecomment-257391790

**Comment by peetucket** [comment]
i think this is no longer necessary?

**Comment by jmartin-sul** [comment]
well... this one upgrades to rails 5 (from 4), and activefedora 8 (from 6)... but probably better to close this and try opening a new PR to do that stuff?  i think there was concern that the AF 6-8 move might require some config changes and some manual testing.

**Comment by jcoyne** [comment]
No longer necessary.

**Pull request closed without merging**